src/Fixer/PhpUnit/PhpUnitDedicateAssertFixer.php

```diff
@@ -169,6 +171,35 @@ public function configure(array $configuration): void
                 'str_contains',
             ]);
         }
+
+        if (PhpUnitTargetVersion::fulfills($this->configuration['target'], PhpUnitTargetVersion::VERSION_9_1)) {
+            $this->fixMap = array_merge($this->fixMap, [
+                'is_readable' => array_merge(
+                    $this->fixMap['is_readable'] ?? [],
+                    [
+                        'negative' => 'assertIsNotReadable',
```

Comment:
to rename `assertNotIsReadable` into `assertIsNotReadable`, it's role of other rule and not this one.